### ISSUE TITLE
CatBoostError: c:/goagent/pipelines/buildmaster/catboost.git/catboost/libs/options/output_file_options.cpp:316: verbose should be a multiple of metric_period, got 1 vs 400

### ISSUE THREAD

**Issue by billy-odera** [catboost/catboost#798]
I have this error when running catboost classifier with below parameters:
```
catboost_params= {"iterations": 1600,
                  "learning_rate": 0.09,
                  "depth": 6,
                  "bagging_temperature": 0.5,
                  "loss_function":'Logloss',
                  "eval_metric": 'CrossEntropy',
                  "metric_period":400,
                  "od_type":'Iter',
                  "od_wait": 200,
                  "random_seed":17,
                  "allow_writing_files":False}
```
and when i fit the model 

```
cb_model = CatBoostClassifier(**catboost_params)
cb_model.fit(X_train, y_train,
             eval_set=(X_valid, y_valid),
             cat_features=cat_features_inds,
             use_best_model=True,
             verbose=True)
```
i get the error 
CatBoostError:c:/goagent/pipelines/buildmaster/catboost.git/catboost/libs/options/output_file_options.cpp:316: verbose should be a multiple of metric_period, got 1 vs 400

**Comment by annaveronika** [comment]
That's a bug, we'll fix it asap.

**Comment by annaveronika** [comment]
For now please set verbose=400

**Comment by annaveronika** [comment]
Btw Logloss and CrossEntropy are really cheep to calculate, so you don't really need metric_period.

**Comment by billy-odera** [comment]
> For now please set verbose=400

so it should be verbose=400 and not verbose=True?

**Comment by annaveronika** [comment]
> so it should be verbose=400 and not verbose=True?
Yes, it should, does it not work?

**Comment by billy-odera** [comment]
> > so it should be verbose=400 and not verbose=True?
> 
> Yes, it should, does it not work?

Thanks Anna, it works

**Comment by annaveronika** [comment]
We'll fix this in https://github.com/catboost/catboost/issues/818

**Comment by songgqin** [comment]
> We'll fix this in #818
CatBoostError: catboost/libs/data/model_dataset_compatibility.cpp:236: Feature 0 from pool must be R.

What's the problem with the above error? However, when I run by it sentence by sentence, it doesn't report the error.
